ts/Series/PointAndFigure/PointAndFigureSeriesDefaults.ts

```diff
@@ -0,0 +1,147 @@
+/* *
+ *
+ *  (c) 2010-2024 Kamil Musialowski
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type PointAndFigureSeriesOptions from './PointAndFigureSeriesOptions';
+
+/* *
+ *
+ *  API Options
+ *
+ * */
+
+/**
+ * The Point and Figure series represents changes in stock price movements,
+ * without focusing on the time and volume. Each data point is created when the
+ * `boxSize` criteria is met. Opposite column of points gets created only when
+ * the `reversalAmount` threshold is met.
+ *
+ * @sample stock/demo/pointandfigure/
+ *         Point and Figure series
+ *
+ * @extends      plotOptions.scatter
+ * @product      highstock
+ * @requires     modules/pointandfigure
+ * @optionparent plotOptions.pointandfigure
+ */
+
+const PointAndFigureSeriesDefaults: PointAndFigureSeriesOptions = {
+    boxSize: '1%',
+    reversalAmount: 3,
+    tooltip: {
+        pointFormat: '<span style="color:{point.color}">\u25CF</span> ' +
+          '<b> {series.name}</b><br/>' +
+          'Close: {point.y:.2f}<br/>',
+        headerFormat: ''
+    },
+    turboThreshold: 0,
+    groupPadding: 0.2,
+    pointPadding: 0.1,
+    pointRange: null,
+    dataGrouping: {
+        enabled: false
+    },
+    markerUp: {
+        symbol: 'cross',
+        lineColor: '#00FF00',
+        lineWidth: 2
+    },
+    marker: {
+        symbol: 'circle',
+        fillColor: 'transparent',
+        lineColor: '#FF0000',
+        lineWidth: 2
+    }
+};
+
+/* *
+ *
+ *  API Options
+ *
+ * */
+
+/**
+ * A `pointandfigure` series. If the [type](#series.pointandfigure.type)
+ * option is not specified, it is inherited from [chart.type](
+ * #chart.type).
+ *
+ * @type      {*}
+ * @extends   series,plotOptions.pointandfigure
+ * @product   highstock
+ * @requires  modules/pointandfigure
+ * @apioption series.pointandfigure
+ */
+
+/**
+ * An array of data points for the series. For the `pointandfigure` series
+ * type, points can be given in the following way:
+ *
+ * 1. An array of arrays with 2 values. In this case, the values correspond
+ *    to `x, close`. Close values are parsed under the hood to create
+ *    point and figure format data points.
+ *    ```js
+ *    data: [
+ *        [1665408600000, 140.42],
+ *        [1665495000000, 138.98],
+ *        [1665581400000, 138.34]
+ *    ]
+ *    ```
```

Comment:
No support for objects like `[{ x, y }]`? Or just missing in docs here?

Reply:
Sorry, I forgot to include it. Tested and works fine with the objects.